crates/rpc-types/src/eth/transaction/mod.rs

```diff
@@ -32,6 +32,7 @@ pub use signature::{Parity, Signature};
 
 /// Transaction object used in RPC
 #[derive(Debug, Default, Clone, PartialEq, Eq, Serialize, Deserialize)]
+#[cfg_attr(all(any(test, feature = "arbitrary"), feature = "std"), derive(arbitrary::Arbitrary))]
```

Comment:
arbitrary should imply std as the dep does not support no_std https://docs.rs/crate/arbitrary/latest/features